docs/developer-docs/multi-chain/chain-key-tokens/ckerc20/overview.mdx

```diff
@@ -0,0 +1,108 @@
+The ledger suite orchestrator makes it possible to handle all ckERC20 tokens in a scalable manner.
+Specifically, it can perform the following actions:
+
+- Spawn a new ICRC ledger suite when adding a new ckERC20 token via an NNS proposal.
+
+- Upgrade all managed ledger suites via an NNS proposal.
+
+- Top up the canisters of all managed ledger suites with cycles.
+## Converting ERC-20 tokens to ckERC20 tokens
+In order to convert ERC-20 tokens to the corresponding ckERC20 tokens, the following steps are carried out:
+
+1. The user submits an Ethereum transaction calling the `approve` function of the ERC-20 smart contract to allow the helper smart contract to withdraw some of the user's funds.
+
+2. The user calls the `deposit` function of the helper smart contract, specifying the ERC-20 contract, how many ERC-20 tokens should be withdrawn from the user's account, and the principal ID that should be credited for the minted ckERC20 tokens. The helper smart contract transfers the specified number of ERC-20 tokens to itself and, if successful, emits an `event` to be consumed by the ckETH minter.
+
+3. The ckETH minter periodically fetches the logs from the helper smart contract, minting ckERC20 tokens for every newly discovered `event`.
+## Converting ckERC20 tokens to ERC-20 tokens
+A user can convert ckERC20 tokens to ERC-20 tokens by following these steps:
+
+1. The user calls the `icrc2_approve` function on the ckETH ledger to approve the ckETH minter to burn some of the user's ckETH tokens as payment for the transaction fees.
+
+2. The user calls the `icrc2_approve` function on the ledger of the given ckERC20 token to approve the minter to burn some of the user's ckERC20 tokens.
+
+3. The user calls the `withdraw_erc20` function on the ckETH minter, specifying the canister ID of the ledger for the given ckERC20 token, the amount to be withdrawn, and the Ethereum destination address. The ckETH minter then issues transactions to burn the fee in ckETH and the specified amount of the ckERC20 token. If both operations succeed, it sends an Ethereum transaction that transfers the specified amount from the helper smart contract to the destination address that the user provided.
+
+More details on both flows are provided on the [ckERC20 GitHub page](https://github.com/dfinity/ic/blob/master/rs/ethereum/cketh/docs/ckerc20.adoc), whereas a detailed description of the ledger suite orchestrator can be found in its [GitHub repository](https://github.com/dfinity/ic/tree/master/rs/ethereum/ledger-suite-orchestrator).
+
+The ledger and index canisters in any ckERC20 ledger suite are regular ICRC-1 compliant canisters. A guide on how to interact with these canisters can be found in the [using the ICRC-1 ledger](/docs/developer-docs/defi/icrc-1/using-icrc1-ledger.mdx) section.
+
+## ckERC20 use-cases
+
+ckERC20 tokens can be used in the same way as other chain-key tokens. Use cases include:
```

Comment:
```suggestion
ckERC20 tokens can be used in the same way as other chain-key tokens. Use-cases include:
```

Reply:
Isn't "use case" more commonly used? It looks like [Merriam-Webster agrees](https://www.merriam-webster.com/dictionary/use%20case).

Reply:
To match the title 'ckERC20 use-cases' above, it should be 'use-cases'. The other Chain Key docs hyphenate it as well. 

As a separate note, I will look into standardizing it across the docs.